### ISSUE TITLE
Would you consider pulling PromiseState out into its own module?

### ISSUE THREAD

**Issue by tkh44** [heroku/react-refetch#162]
I think it could be very handy for devs who can't use react-refetch, but find utility in PromiseState.

**Comment by ryanbrainard** [comment]
I had considered it as some point, but didn't know if there would be interest. I guess there is :) Not sure how soon I'll get it it, but in the meantime you can do `import {PromiseState} from 'react-refetch'` to just import the `PromiseState`

**Comment by tkh44** [comment]
Yeah, I'm doing it that way right now. Just thought I would make an official request.
